app/assets/javascripts/discourse/app/controllers/user-activity-bookmarks.js

```diff
@@ -30,16 +40,36 @@ export default Controller.extend({
       this.set("searchTerm", this.q);
     }
 
+    if (!this._boundClick) {
+      schedule("afterRender", () => {
+        // TODO(martin): This should be pulled out into a bookmark-list component,
```

Comment:
We should definitely do this TODO sooner rather than later. We should not be manipulating the dom in a controller.

Reply:
Yes I think Martin is working on this.